### PR TITLE
Update README and add script for publishing to GH pages

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Connects to #31

change-type: patch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,22 +45,26 @@ npm install
...
+The interactive storybook can be published by running:
+
+```sh
+npm run publish-storybook
 ```
 
-### Publishing
+## Testing
 
 ```
```
Comment (thgreasi) [inline README.md]: How about adding an `sh` here as well?

File: src/stories/PineTypes.js [hunk 2]
```diff
@@ -1,81 +0,0 @@
-import { h, Component } from 'preact'
```
Comment (thgreasi) [inline src/stories/PineTypes.js]: I wouldn't expect to see such a deletion in this PR.

File: .scripts/publish_storybook.sh [hunk 3]
```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```
Comment (thgreasi) [inline .scripts/publish_storybook.sh]: Could we use something like [gh-pages](https://github.com/tschaub/gh-pages) for this?

File: src/stories/PineTypes.js (hunk 2 above)
Comment (LucianBuzzo) [reply src/stories/PineTypes.js]: Yeah it got commited by mistake I think

File: .scripts/publish_storybook.sh (hunk 3 above)
Comment (LucianBuzzo) [reply .scripts/publish_storybook.sh]: This looks good, I'm going to create a separate issue for addressing this in the future.

Comment (LucianBuzzo) [reply .scripts/publish_storybook.sh]: https://github.com/resin-io-modules/resin-components/issues/39

**Pull request merged**